renderdoc/driver/gl/gl_driver.h

```diff
@@ -279,6 +279,11 @@ class WrappedOpenGL : public IFrameCapturer
     // bound to the second.
     GLuint renderbufferReadTex;
     GLuint renderbufferFBOs[2];
+
+    // since compressed textures cannot be read back on GLES we have to store them during the
+    // uploading
+    // target -> level -> data
+    map<int, map<int, vector<byte> > > compressedData;
```

Comment:
Is the map keyed on target needed? It's illegal to re-bind a texture from one target to another, the target is basically immutable for the lifetime of a texture (ie. you can't take a TEXTURE_2D and then make it a TEXTURE_2D_ARRAY or TEXTURE_3D). I think you only need to separate by level.